### ISSUE TITLE
HTTP and WebSockets?

### ISSUE THREAD

**Issue by emiltin** [rsmp-nordic/rsmp_core#61]
Was HTTP and WebSockets considering when choosing raw TCP/IP for RSMP?

The site would run an http server, responding to commands. A supervisor system could subscribe to data and events using WebSockets. The site would push data when available.

https://en.wikipedia.org/wiki/WebSocket

**Comment by otterdahl** [comment]
I don't think so. The raw TCP/IP was chosen back in early 2011 with the release of RSMP 3.0 At that time the RFC for WebSockets was not released yet.
